### PR TITLE
Change blindsign flow according to guideline.

### PR DESCRIPTION
**Pull request by ajinkyaraj-23**
Remove blindsign button from stax

# Checklist

- [ ] App update process has been followed 
- [ ] Target branch is `develop` 
- [ ] Application version has been bumped

### CODE REVIEW

File: app/src/ui_stream_nbgl.c [hunk 1]
```diff
@@ -90,30 +90,36 @@ start_blindsign(void)
...
-        if (!N_settings.blindsigning) {
-            toggle_blindsigning();
-        }
-        nbgl_useCaseStatus("BLIND SIGNING\nENABLED", true, blindsign_splash);
-    } else {
         tz_reject_ui();
+    } else {
+        char blindsign_msg[]
+            = "Transaction could not be decoded correctly. \nLearn More: "
+              "tinyurl.com/ledger-Tezos\nERROR:                 ";
```
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/ui_stream_nbgl.c:113]: Fixed it. now it leads to our repo in github.

File: app/src/ui_stream_nbgl.c [hunk 2]
```diff
@@ -90,30 +90,36 @@ start_blindsign(void)
...
+        const int BLINDSIGN_MSG_ERROR_OFFSET = 90;
```
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/ui_stream_nbgl.c:114]:
```suggestion
       char blindsign_msg[150]
            = "Transaction could not be decoded correctly. \nLearn More: "
              "tinyurl.com/ledger-Tezos\nERROR:";
       memcpy(blindsign_msg + sizeof(blindsign_msg), global.error_code, ERROR_CODE_SIZE);
```

[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/ui_stream_nbgl.c:114]: How does this sound?

[the remark this answers is not in the document]
Comment (spalmer25) [reply app/src/ui_stream_nbgl.c:114]: If that's not possible, let's do that.

[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply app/src/ui_stream_nbgl.c]: Done.

File: tests/integration/touch/test_basic.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 # Copyright 2023 Trilitech <contact@trili.tech>
+import os
```
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply tests/integration/touch/test_basic.py]: Removed.

**Pull request merged**